libiocage/lib/Release.py

```diff
@@ -310,75 +369,129 @@ def update(self):
         dataset.snapshot(snapshot_name, recursive=True)
 
         jail = libiocage.lib.Jail.Jail({
-            "uuid"              : str(uuid.uuid4()),
-            "basejail"          : False,
-            "allow_mount_nullfs": "1",
-            "release"           : self.name,
-            "securelevel"       : "0"
-        },
+                "uuid"              : str(uuid.uuid4()),
+                "basejail"          : False,
+                "allow_mount_nullfs": "1",
+                "release"           : self.name,
+                "securelevel"       : "0"
+            },
+            new=True,
```

Comment:
Why are we creating a new jail for `update`?

Reply:
We're doing the same for freebsd. It doesn't cost much to start a jail from a release and both freebsd-update.sh and hbsd-update support mechanisms that allow us to do that. On FreeBSD it's even more elegant because it can fetch updates for offline usage.

Anything wrong with that?